### ISSUE TITLE
Allow Node for `text` prop

### ISSUE THREAD

**Issue by iowillhoit** [ShinyChang/React-Text-Truncate#32]
I have a Highlighter component that highlights substrings based off of a `search` prop. It would be great if I could pass my Highlighter component to your `text` prop so that the string could be both truncated and highlighted. 

Example:

``` jsx
const desc = <Highlighter search={this.props.search}>{this.props.description}</Highlighter>;

return (
    <Truncate
        line={2}
        truncateText="…"
        text={desc}
    />
);
```

**Comment by francisngo** [comment]
Looking to find out how to do this too.

**Comment by ShinyChang** [comment]
Will looking for solution :)

**Comment by ShinyChang** [comment]
Fixed in v0.12.0
